pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -652,16 +652,20 @@ func (ds *dockerService) makeSandboxDockerConfig(c *runtimeapi.PodSandboxConfig,
 	}
 
 	// Set security options.
-	securityOpts, err := ds.getSecurityOpts(c.GetLinux().GetSecurityContext().GetSeccompProfilePath(), securityOptSeparator)
-	if err != nil {
-		return nil, fmt.Errorf("failed to generate sandbox security options for sandbox %q: %v", c.Metadata.Name, err)
-	}
+	securityOpts := ds.getSandBoxSecurityOpts(securityOptSeparator)
 	hc.SecurityOpt = append(hc.SecurityOpt, securityOpts...)
 
 	applyExperimentalCreateConfig(createConfig, c.Annotations)
 	return createConfig, nil
 }
 
+func (ds *dockerService) getSandBoxSecurityOpts(separator rune) []string {
```

Comment:
I'd like to put this behind a feature gate, so it's easier to roll back if needed. It might be good to start in alpha too, to enable more time for soak testing.

Reply:
I will add the feature gate today. Are you OK with the name and default values as per below?

```golang
PodSandboxSeccomp = false
```